app/process/handler/ethereum/handler.go

```diff
@@ -19,28 +19,37 @@ package ethereum
 import (
 	"github.com/limechain/hedera-eth-bridge-validator/app/domain/service"
 	burn_event "github.com/limechain/hedera-eth-bridge-validator/app/model/burn-event"
+	lock_event "github.com/limechain/hedera-eth-bridge-validator/app/model/lock-event"
 	"github.com/limechain/hedera-eth-bridge-validator/config"
 	log "github.com/sirupsen/logrus"
 )
 
 type Handler struct {
-	service service.BurnEvent
-	logger  *log.Entry
+	lockService service.LockEvent
+	burnService service.BurnEvent
+	logger      *log.Entry
 }
 
-func NewHandler(service service.BurnEvent) *Handler {
+func NewHandler(burnService service.BurnEvent, lockService service.LockEvent) *Handler {
 	return &Handler{
-		service: service,
-		logger:  config.GetLoggerFor("Scheduled Transaction Handler"),
+		lockService: lockService,
+		burnService: burnService,
+		logger:      config.GetLoggerFor("Scheduled Transaction Handler"),
 	}
 }
 
 func (sth Handler) Handle(payload interface{}) {
```

Comment:
Everytime we are receiving lock event, we will be logging error `Could not cast payload [%s] to [burnEvent]` and then processing the lock. I do not think that this is okay in terms of logs. Lets change the logic so that we try to parse the event in one of the 2 payloads or introduce a brand new handler for the lock event